container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/consts/consts.go

```diff
@@ -31,6 +31,8 @@ const (
 	//The Docker network name where all the containers in the engine and logs service context will be added
 	NameOfNetworkToStartEngineAndLogServiceContainersIn = "bridge"
 	HttpApplicationProtocol                             = "http"
+
+	GitHubAuthStorageDirPath = "/kurtosis-data/github-auth/"
```

Comment:
This directory is meant to store the GitHub auth in the enclave data directory of the `APIC` - but this const should really refactored to be whatever the enclave data directory is instead of manually hardcoded. this would require creating a top level `consts` or changing where the `enclave` data directory is determined (right now passed into `backend.CreateAPIContainer` from a const in `api_container_launcher`